docs/src/concepts.md

```diff
@@ -3,12 +3,28 @@
 ## Acquisition Tools for Electrophysiology
 
 ### Neuropixels Probes
-Neuropixels probes were developed by a collaboration between HHMI Janelia, industry partners, and others<sup>[1](#references)</sup>. Since their initial release in October 2018, 300 labs have ordered 1200 probes. Since the rollout of Neuropixels 2.0 in October 2020, IMEC has been shipping 100+ probes monthly (correspondence with Tim Harris).
 
-Neuropixels probes offer 960 electrode sites along a 10mm long shank, with 384 recordable channels per probe that can record hundreds of units spanning multiple brain regions (Neuropixels 2.0 version is a 4-shank probe with 1280 electrode sites per shank). Such large recording capacity has offered tremendous opportunities for the field of neurophysiology research, yet this is accompanied by an equally great challenge in terms of data and computation management.
+Neuropixels probes were developed by a collaboration between HHMI Janelia, industry
+partners, and others[^1]. Since their initial release in October
```

Comment:
Is this citation link supposed to link to anything, or does it just provide reference to the citations located elsewhere?

Reply:
@sidhulyalkar `[^1]` is the format for the mkdocs footnotes extension. The citation is provided on line 19 below. [Example deployment](https://datajoint.com/docs/elements/element-electrode-localization/0.1/concepts/)